github/actions_runners.go

```diff
@@ -45,6 +45,60 @@ func (s *ActionsService) ListRunnerApplicationDownloads(ctx context.Context, own
 	return rads, resp, nil
 }
 
+// GenerateJITConfigRequest specifies body parameters to GenerateJITConfig.
+type GenerateJITConfigRequest struct {
+	Name          string  `json:"name"`
+	RunnerGroupID int64   `json:"runner_group_id"`
+	WorkFolder    *string `json:"work_folder,omitempty"`
+
+	// Labels represents the names of the custom labels to add to the runner.
+	// Minimum items: 1. Maximum items: 100.
+	Labels []string `json:"labels"`
+}
+
+// JITRunnerConfig represents encoded JIT configuration that can be used to bootstrap a self-hosted runner.
+type JITRunnerConfig struct {
+	EncodedJITConfig *string `json:"encoded_jit_config,omitempty"`
+}
+
+// GenerateOrganizationJITConfig generate a just-in-time configuration for an organization.
```

Comment:
What do you think about naming this one "GenerateOrgJITConfig"
and the one below "GenerateRepoJITConfig"?
I think there might be a little less confusion if these are explicitly "Org" and "Repo".
Thoughts?

Reply:
That would also work. I tried to keep it consistent with `CreateRegistrationToken/CreateOrganizationRegistrationToken`, but I wouldn't mind renaming.